module/solutions/helper.js

```diff
@@ -2864,6 +2971,217 @@ module.exports = class SolutionsHelper {
 		})
 	}
 
+	// static targetedSolutions(
```

Comment:
@Prajwal17Tunerlabs We should remove unused code

Reply:
removed commented code.